quickwit/quickwit-proto/protos/quickwit/metastore.proto

```diff
@@ -216,8 +221,8 @@ message DeleteTask {
 }
 
 message DeleteQuery {
-  // Index ID.
-  string index_uid = 1;
+  // Index UID.
+  IndexUid index_uid = 1;
```

Comment:
Now, we have a typed `IndexUid` in protobuf messages. Yay!
Now, we have to deal with an option. Nay!

Reply:
I think that's a breaking change and would technically require to change the field number. I don't know how much we care about that though.